scrnaseq.Rmd

```diff
@@ -1339,72 +1341,76 @@ p = p + patchwork::plot_annotation(title="Cell cycle phases")
+## UMAP coloured by S phase
 ```{r cellCycleEffect_umap_s}
 p = Seurat::FeaturePlot(sc, features="S.Score", pt.size=1, min.cutoff="q1", max.cutoff="q99", label=TRUE, cols=c("lightgrey", param$col)) + 
   AddStyle(title="UMAP, cells coloured by S phase")
 p
 ```
 
-### UMAP coloured by G2/M phase
+## UMAP coloured by G2/M phase
 ```{r cellCycleEffect_umap_g2m}
 p = Seurat::FeaturePlot(sc, features="G2M.Score", pt.size=1, min.cutoff="q1", max.cutoff="q99", cols=c("lightgrey", param$col), label=TRUE) + 
   AddStyle(title="UMAP, cells coloured by G2M phase")
 p
 ```
 
-### UMAP coloured by the difference between S and G2/M phase
+## UMAP coloured by the difference between S and G2/M phase
 ```{r cellCycleEffect_umap_ccdiff}
 p = Seurat::FeaturePlot(sc, features="CC.Difference", pt.size=1, min.cutoff="q1", max.cutoff="q99", cols=c("lightgrey", param$col), label=TRUE) +
   AddStyle(title="UMAP, cells coloured by CC.Difference")
 p
 ```
 
-## Cluster QC  {.tabset}
+# Cluster QC  {.tabset}
 Do cells in individual clusters have particularly high counts, detected genes or mitochondrial content?
 
-### Number of counts 
+## Number of counts 
 ```{r clusterQC_nCount_featurePlot}
-p1 = Seurat::FeaturePlot(sc, features="nCount_RNA", cols=c("lightgrey", param$col), label=TRUE) + 
-  AddStyle(title="Feature plot")
+p1 = Seurat::FeaturePlot(sc, features="nCount_RNA", label=TRUE) + 
+  AddStyle(title="Feature plot") + 
+  scale_colour_gradient(low="lightgrey", high=param$col, trans="log10")
 
 p2 = ggplot(sc[[]], aes(x=seurat_clusters, y=nCount_RNA, fill=seurat_clusters)) + 
   geom_violin(scale="width") + 
-  AddStyle(title="Violin plot", fill=param$col_clusters,
-           xlab="Cluster", legend_position="none")
+  AddStyle(title="Violin plot (log10 scale)", fill=param$col_clusters,
+           xlab="Cluster", legend_position="none") + 
+  scale_y_log10()
 
 p = p1 | p2 
-p = p + patchwork::plot_annotation(title="Summed raw counts (nCount_RNA)")
+p = p + patchwork::plot_annotation(title="Summed raw counts (nCount_RNA, log10 scale)")
 p
 ```
 
-### Number of features
+## Number of features
 ```{r clusterQC_nFeature_featurePlot}
-p1 = Seurat::FeaturePlot(sc, features="nFeature_RNA", cols=c("lightgrey", param$col), label=TRUE) + 
-  AddStyle(title="Feature plot")
+p1 = Seurat::FeaturePlot(sc, features="nFeature_RNA", label=TRUE) + 
+  AddStyle(title="Feature plot") + 
+  scale_colour_gradient(low="lightgrey", high=param$col, trans="log10")
 
 p2 = ggplot(sc[[]], aes(x=seurat_clusters, y=nFeature_RNA, fill=seurat_clusters)) + 
   geom_violin(scale="width") + 
   AddStyle(title="Violin plot", fill=param$col_clusters,
-           xlab="Cluster", legend_position="none")
+           xlab="Cluster", legend_position="none") + 
+  scale_y_log10()
 
 p = p1 | p2 
-p = p + patchwork::plot_annotation(title="Number of features with raw count > 0 (nFeature_RNA)")
+p = p + patchwork::plot_annotation(title="Number of features with raw count > 0 (nFeature_RNA, log10 scale)")
```

Comment:
I think the "> 0" is not neccessary here.

Reply:
I like it, can we leave it?

Reply:
hmm ok